spec/dummy/app/assets/config/manifest.js

```diff
@@ -1,2 +1,3 @@
 //= link social_networking.js
 //= link social_networking.css
+//= link social_networking/_profile_icon_admin.png
```

Comment:
Were you able to test that this icon loaded locally in a host application? I was having trouble loading image resources through the engine manifests. I had to add the image paths to the host application manifest for several TFDEngine images.